src/main/scala/org/ergoplatform/settings/Constants.scala

```diff
@@ -11,17 +11,26 @@ import scorex.core.{ModifierTypeId, NodeViewModifier}
 
 object Constants {
   val HashLength: Int = 32
+
   val CoinsInOneErgo: Long = 1000000000
+
   val MaxTarget: BigInt = BigInt(1, Array.fill(HashLength)((-1).toByte))
   val InitialDifficulty: Difficulty = BigInt(1)
   val InitialNBits: Long = RequiredDifficulty.encodeCompactBits(InitialDifficulty)
   val ModifierIdSize: Int = HashLength
-  // Max total computation cost of a block. todo: move to config
-  val MaxBlockCost: Long = 1000000
-  // Max cost of coinbase transaction. todo calculate?
+
+  // Max cost of coinbase transaction. todo calculate? todo: do we need this constant
   val CoinbaseTxCost: Int = 10000
-  // Max size of transactions section of a block. todo: move to config
-  val MaxBlockSize: Int = 512 * 1024
+
+  val BlocksPerHour = 30
+
+  //For how many blocks a box could be put into the state with no paying.
+  //4 years
+  val StoragePeriod: Int = 4 * 365 * 24 * BlocksPerHour
+
+  val StorageContractCost: Long = 50
```

Comment:
We're going to allow to change cost for all opcodes, so it should be placed to `Parameters`.